src/app/components/navbar/navbar.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-navbar',
+  templateUrl: './navbar.component.html',
+  styleUrls: ['./navbar.component.css']
+})
+export class NavbarComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
```

Comment:
Skal vi ikke have router logic med her?

Reply:
router logic er i html

Reply:
Mit gæt ville være at det er mere "rigtigt" at bruge angular router logik i stedet for <a> tags.